swagger_server/itm/itm_history.py

```diff
@@ -0,0 +1,67 @@
+import json
+import os
+from typing import Union
+
+class ITMHistory:
+    """
+    Class for managing ADM action history.
+    """
+
+    def __init__(self):
+        """
+        Initialize an instance of ITMSupplies.
+        """
+        self.history = []
+        self.filepath = 'itm_db_local_output/'
```

Comment:
Is the trailing '/' necessary?  It's an os specific path separator.  It may not matter since we use os.path.join on it later and that has some fancy join logic.  Otherwise, it should be os.sep.

Reply:
It turns out the `/` is necessary.  Changed to `os.sep`.